project/src/demo/ui/chat/mood-demo.gd

```diff
@@ -68,5 +68,5 @@ func _input(event: InputEvent) -> void:
 			KEY_V: $Creature.play_mood(ChatEvent.Mood.WAVE1)
 			KEY_B: $Creature.play_mood(ChatEvent.Mood.YES0)
 			KEY_N: $Creature.play_mood(ChatEvent.Mood.YES1)
-			KEY_SPACE: $Creature.feed(Playfield.FOOD_COLORS[0])
+			KEY_SPACE: $Creature.feed(FoodColors.FOOD_COLORS[0])
```

Comment:
Document the purpose of this new 'FoodColors' class in the commit comment

Reply:
Added comment `Extracted FoodColors constants class to avoid cyclic reference ...`